encoding/encoding.go

```diff
@@ -0,0 +1,60 @@
+// Copyright 2019 Decipher Technology Studios
```

Comment:
Out of curiosity, what year do you think this is?

Reply:
Ahhh, dang. You've exposed my blatantly lazy copy-paste maneuver. My apologies.